app/view-plants/page.tsx

```diff
@@ -319,12 +325,52 @@ export default function Page() {
 
   const plantPluralityString = selectedPlants.length > 1 ? 'Plants' : 'Plant';
 
+  // close plant card key when clicking outside, even on info button
+  const handleClickOutside = (event: MouseEvent) => {
+    if (
+      cardKeyRef.current &&
+      !cardKeyRef.current.contains(event.target as Node) &&
+      infoButtonRef.current &&
+      !infoButtonRef.current.contains(event.target as Node)
+    ) {
+      setIsCardKeyOpen(false);
+    }
+  };
+
+  // handle clicking outside PlantCardKey to close it if open
+  useEffect(() => {
+    if (isCardKeyOpen) {
+      document.addEventListener('mousedown', handleClickOutside);
+    } else {
+      document.removeEventListener('mousedown', handleClickOutside);
+    }
+
+    return () => {
+      document.removeEventListener('mousedown', handleClickOutside);
+    };
+  }, [isCardKeyOpen]);
+
   return (
     <div id="plantContent">
       <TopRowContainer>
-        <H1 $color={COLORS.shrub} $fontWeight={500}>
-          View Plants
-        </H1>
+        <Flex $direction="row" $gap="10px" $align="center">
+          <H1 $color={COLORS.shrub} $fontWeight={500}>
+            View Plants
+          </H1>
+          <div style={{ position: 'relative' }}>
+            <InfoButton
+              onClick={() => setIsCardKeyOpen(!isCardKeyOpen)}
+              ref={infoButtonRef}
+            >
+              <Icon type="info" />
```

Comment:
what if we tried  `<Icon type="info" as='button' onClick={} ref={} />` directly instead of defining a new button style?